src/util/theme.py

```
@@ -11,7 +13,32 @@
 logger = logging.getLogger(__name__)
 
 
-class Theme():
+def _noneIfNoFile[**P, R](
+    fun: Callable[Concatenate[Theme, P], R],
```

Ruff (0.14.10)

Undefined name `Theme`

(F821)